addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -460,12 +468,16 @@ QUnit.test('new messages separator on posting message', async function (assert)
 
     document.querySelector('.o_ComposerTextInput_textarea').focus();
     await afterNextRender(() => document.execCommand('insertText', false, "hey !"));
-    await afterNextRender(() =>
-        document.querySelector('.o_Composer_buttonSend').click()
-    );
-    assert.containsOnce(
+    await afterNextRender(() => {
+        // need to remove focus from text area to avoid channel_seen
+        document.querySelector('.o_Composer_buttonSend').focus();
+        document.querySelector('.o_Composer_buttonSend').click();
+
+    });
+    assert.containsN(
         document.body,
         '.o_Message',
+        2,
         "should have the message current partner just posted"
```

Comment:
Don't forget to update the assertion comment, which should more closely match the assertion itself.
E.g. _"should not display 2 messages in the conversation, after posting a new message (initial message + newly posted)"_.